portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.data;
+
+import java.time.Duration;
+
+/** Models a task. It can be used for both creation and import flow. */
+public final class Task {
+
+  private final String name;
+  private final String description;
+  private final Duration durationMinute;
+  private final TaskPriority priority;
+
+  // TODO(raulcruise): Add an ID field if necessary.
+
+  // The constructor will make sure that all necessary parameters are passed in,
+  // and populate each class variable appropriately.
+  public Task(String name, String description, Duration durationMinute, Integer priority) {
```

Comment:
Let's use javadoc style.

/**
 * ...
 * ...
 */

Also, please indicate in the constructor which fields can be optional or null. If not, clarify that none of the fields should be null.

Reply:
Done.